tests/integration/cattletest/core/test_docker.py

```diff
@@ -637,9 +637,9 @@ def test_container_fields(docker_client, super_client):
     assert set(c.data['dockerInspect']['HostConfig']['CapAdd']) == set(caps)
     assert set(c.data['dockerInspect']['HostConfig']['CapDrop']) == set(caps)
     actual_dns = c.data['dockerInspect']['HostConfig']['Dns']
-    assert set(actual_dns) == set(['8.8.8.8', '1.2.3.4'])
+    assert set(actual_dns) == set(['8.8.8.8', '1.2.3.4', '169.254.169.250'])
```

Comment:
no...we still put our dns there, and its set on the container, and our dns will be used. We just don't propagate it to dns config yml